package.json

```diff
@@ -3,20 +3,21 @@
   "type": "module",
   "version": "0.0.1",
   "scripts": {
-    "dev": "dotenv -e .env.development -- astro dev",
-    "start": "dotenv -e .env.development -- astro dev --host",
+    "dev": "dotenv -e .env -- astro dev",
+    "start": "dotenv -e .env -- astro dev --host",
```

Comment:
Any reason to keep `dotenv`?

Reply:
yes... otherwise, problems with the prisma DATABASE_URL env

Reply:
Prisma should be working with `.env` file well. The issues you've mentioned are likely related to `.env.development` and `.env.other.stuff`. [Might help](https://stackoverflow.com/questions/67796217/prisma-getting-environment-variable-not-found-error-message-when-running-graph).